hack/lib/golang.sh

```diff
@@ -232,9 +232,23 @@ kubeedge::golang::build_binaries() {
   for bin in ${binaries[@]}; do
     echo "building $bin"
     local name="${bin##*/}"
-    set -x
-    go build -o ${KUBEEDGE_OUTPUT_BINPATH}/${name} -gcflags="${gogcflags:-}" -ldflags "${goldflags:-}" $bin
-    set +x
+    # if env BUILD_WITH_CONTAINER exist, we build with docker images
+    set +u
+    if [ ! -z ${BUILD_WITH_CONTAINER} ]; then
+      KUBEEDGE_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/../.." && pwd -P)"
+      MOUNTPATH="/kubeedge"
+      KUBEEDGE_BUILD_IMAGE="kubeedge/build-tools"
```

Comment:
Is it possible to place them to [build_with_container.sh](https://github.com/kubeedge/kubeedge/pull/3756/files#diff-40fbba595324eaaaf74f799fa28f7ee95df1d1f4b603483eceaec5983dfa2407)?

Reply:
fixed, PTAL 😄